apps/backend/src/domain/location/location.ts

```diff
@@ -0,0 +1,24 @@
+import { EntityError } from '../errors/entity.error';
+
+export default class Location {
+  private value: {
+    latitude: number;
+    longitude: number;
+  };
+
+  constructor(longitude: number, latitude: number) {
+    if (typeof latitude !== 'number') {
```

Comment:
Latitude and longitude have natural constraints (latitude: -90 to 90, longitude: -180 to 180). Adding range checks can prevent invalid coordinates.